homeassistant/components/amberelectric/strings.json

```diff
@@ -1,25 +1,67 @@
 {
   "config": {
+    "error": {
+      "invalid_api_token": "Invalid API key",
+      "no_site": "No site provided",
+      "unknown_error": "Unexpected error"
+    },
     "step": {
-      "user": {
-        "data": {
-          "api_token": "[%key:common::config_flow::data::api_token%]",
-          "site_id": "Site ID"
-        },
-        "description": "Go to {api_url} to generate an API key"
-      },
       "site": {
         "data": {
           "site_id": "Site NMI",
           "site_name": "Site Name"
         },
         "description": "Select the NMI of the site you would like to add"
+      },
+      "user": {
+        "data": {
+          "api_token": "API token",
+          "site_id": "Site ID"
+        },
+        "description": "Go to {api_url} to generate an API key"
+      }
+    }
+  },
+  "services": {
+    "get_forecasts": {
+      "name": "Get price forecasts",
+      "description": "Retrieve price forecasts",
+      "fields": {
+        "site_id": {
+          "description": "The site to get forecasts from.",
+          "name": "Site"
+        },
+        "channel_type": {
+          "description": "The channel to get forecasts for.",
+          "name": "Channel type"
+        }
       }
+    }
+  },
+  "exceptions": {
+    "integration_not_found": {
+      "message": "Integration \"{target}\" not found in registry."
     },
-    "error": {
-      "invalid_api_token": "[%key:common::config_flow::error::invalid_api_key%]",
-      "no_site": "No site provided",
-      "unknown_error": "[%key:common::config_flow::error::unknown%]"
+    "not_loaded": {
+      "message": "{target} is not loaded."
+    },
+    "channel_not_found": {
+      "message": "There is no {channel_type} channel at this site."
+    }
+  },
+  "selector": {
+    "channel_type": {
+      "options": {
+        "general": "General",
+        "controlled_load": "Controlled load",
+        "feed_in": "Feed in"
+      }
+    }
+  },
+  "issues": {
+    "deprecated_forecast_sensor": {
+      "description": "The forecast sensor `{entity}` is deprecated and is used in the following automations or scripts:\n{items}\n\nThe get_forecasts action is available and should be used going forward; Please use it on the above automations or scripts to fix this issue.",
+      "title": "Deprecated forecast sensor detected in some automations or scripts"
```

Comment:
interesting how sometimes its alphabetic and sometimes its not. I personally kinda do prefer to have the name first and then the description, so in that regard, I am not a big alphabetical fan